src/web/static.gleam

```
@@ -84,35 +94,39 @@ fn get_asset(
   }
 }
 
-pub fn collect_assets(from base: String) -> Assets {
+pub fn collect_assets(
+  from base: String,
+  types types: Map(String, String),
+) -> Assets {
   map.from_list({
     use relative_path <- list.filter_map(path.wildcard(base, "**"))
```

How should `Error(Nil)` be handled?

- Silently ignore (like now, probably not the best choice)
- Crash
- Log
- Return a Result